cross-chain/arbitrum/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "cross-chain-arbitrum",
+  "version": "0.0.1",
+  "description": "Cross chain Arbitrum",
+  "license": "MIT",
+  "files": [
+    "artifacts/",
+    "build/contracts/",
+    "contracts/",
+    "!contracts/hardhat-dependency-compiler",
+    "!**/test/",
+    "deploy/",
+    "export/",
+    "tasks/",
+    "export.json"
+  ],
+  "scripts": {
```

Comment:
Can we order `scripts` alphabetically?

Reply:
Done [d7de7a3](https://github.com/keep-network/tbtc-v2/pull/554/commits/d7de7a311c53cf8d9b68e861621c7dc6daa981d4)